content/en/docs/making-requests.md

```diff
@@ -207,133 +257,138 @@ if err := decoder.Decode(&myStruct); err != nil {
-                // Add the objects to the channel for each page
-                for _, object := range page.Contents {
-                    keysCh <- fmt.Sprintf("%s:%s", *params.Bucket, *object.Key)
-                }
-                return true
-            },
-        )
-        if err != nil {
-            fmt.Println("Error listing", *params.Bucket, "objects:", err)
-        }
-    }(params)
+
+client := s3.NewFromConfig(cfg)
+
+type result struct {
+    Output *s3.PutObjectOutput
+    Err    error
 }
+
+results := make(chan result, 2)
+
+var wg sync.WaitGroup
+wg.Add(2)
+
+go func() {
+defer wg.Done()
+    output, err := client.PutObject(context.Background(), &s3.PutObjectInput{
+        Bucket: aws.String("my-bucket"),
+        Key:    aws.String("foo"),
+        Body: bytes.NewReader([]byte("foo body content")),
+    })
+    results <- result{Output: output, Err: err}
+}()
+
 go func() {
-    wg.Wait()
-    close(keysCh)
+    defer wg.Done()
+    output, err := client.PutObject(context.Background(), &s3.PutObjectInput{
+        Bucket: aws.String("my-bucket"),
+        Key:    aws.String("bar"),
+        Body: bytes.NewReader([]byte("bar body content")),
+    })
+    results <- result{Output: output, Err: err}
 }()
-for key := range keysCh {
-    // Print out each object key as its discovered
-    fmt.Println(key)
+
+wg.Wait()
+
+close(results)
+
+for result := range results {
+    if result.Err != nil {
+        log.Printf("error: %v", result.Err)
+        continue
+    }
+    fmt.Printf("etag: %v", aws.ToString(result.Output.ETag))
 }
 ```
 
-## Using Pagination Methods
+## Using Operation Paginators
 
 Typically, when you retrieve a list of items, you might need to check
-the output for a token or marker to confirm whether AWS returned all
-results from your request. If present, you use the token or marker to
-request the next set of results. Instead of managing these tokens or
-markers, you can use pagination methods provided by the SDK.
-
-Pagination methods iterate over a list operation until the method
-retrieves the last page of results or until the callback function
-returns `false`. The names of these methods use the following pattern:
-*OperationName*\ `Pages`. For example, the pagination method for the
-|S3| list objects operation (`ListObjects`) is `ListObjectPages`.
-
-The following example uses the `ListObjectPages` pagination method to
-list up to three pages of object keys from the `ListObject`
-operation. Each page consists of up to 10 keys, which is defined by
-the `MaxKeys` field.
+the output for a token or marker to confirm whether an AWS returned all results from your request.
+If present, you use the token or marker to request the next page of results. Instead of managing these tokens or
+markers, you can use the service package's available paginator types.
+
+Paginator helpers are available for support service operations. Constructing an operation paginator is using operations
```

Comment:
I think the sentence "Paginator helpers are available for support service operations." can go entirely and the one after that needs work; I am having trouble understanding what it means.

Reply:
Good catch, clearly I must have walked away and came back to write gibberish :)